common/settings.h

```diff
@@ -163,6 +163,9 @@ struct Settings {
   // Selects the SkParagraph implementation of the text layout engine.
   bool enable_skparagraph = false;
 
+  // Selects the DisplayList for storage of rendering operations.
+  bool enable_display_list = true;
```

Comment:
Is this on by default just for purposes of making the presubmits use the display list stuff? I think we should land this with off by default and in another patch, flip the switch.

Reply:
Correct. The next commit would be to turn the flag off and remove the WIP.

Reply:
I just pushed a commit changing the flag to false by default and removed the WIP designation.